src/components/shared/ButtonWithPanel/ArrowPanel.css

```diff
@@ -65,38 +77,45 @@
   }
 }
 
-.arrowPanelArrow {
-  position: absolute;
-  top: calc(var(--internal-offset-from-top) * -1);
-  right: 0;
-  left: 0;
+/**
+ * This element clips the arrow to its triangle shape.
+ */
+.arrowPanelArrowContainer {
+  position: relative;
   overflow: hidden;
-  height: var(--internal-offset-from-top);
+  height: var(--internal-arrow-clip-height);
+  flex-shrink: 0;
```

Comment:
optional nit: I'm more used to `flex: none` that's IMO more explicit.

Reply:
Sounds good.